Dockerfile

```diff
@@ -2,9 +2,12 @@ FROM crystallang/crystal:latest-alpine AS builder
 RUN apk update && apk upgrade && apk add sqlite-static
 WORKDIR /build/
 ARG version
-RUN git clone --branch ${version:-v2.0.0-10} --depth 1 https://github.com/toddsundsted/ktistec .
+RUN git clone --branch ${version:-v2.0.0} --depth 1 https://github.com/toddsundsted/ktistec .
 RUN shards update && shards install --production
 RUN crystal build src/ktistec/server.cr --static --no-debug --release
+RUN apk add npm
+RUN npm install --save-dev webpack
+RUN npm run build
```

Comment:
this step is in fact required if we are not pulling from `version=dist`, i.e. if we want to track the version numbers